src/components/TreeView.js

```diff
@@ -56,6 +96,14 @@ function RowValue(props) {
     separator = ': ';
   }
 
+  if (node.type === 'signatures') {
+    return formatSignatureCheckState(node, separator)
+  }
+
+  if (node.type === "signature") {
```

Comment:
Use `position` prop here too but use regexp to check if you are in `TransactionEnvelope.signatures[x].signature`.